app/kubernetes-ingress-controller/1.0.x/guides/configuring-fallback-service.md

```diff
@@ -43,9 +43,48 @@ For the purpose of this guide, we will setup an [httpbin](https://httpbin.org)
 service in the cluster and proxy it.
 
 ```bash
-$ kubectl apply -f https://bit.ly/k8s-httpbin
-service/httpbin created
-deployment.apps/httpbin created
+$ echo '
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fallback-svc
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: fallback-svc
+  template:
+    metadata:
+      labels:
+        app: fallback-svc
+    spec:
+      containers:
+      - name: fallback-svc
+        image: hashicorp/http-echo
+        args:
+        - "-text"
+        - "This is not the path you are looking for. - Fallback service"
+        ports:
+        - containerPort: 5678
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fallback-svc
+  labels:
+    app: fallback-svc
+spec:
+  type: ClusterIP
+  ports:
+  - port: 80
+    targetPort: 5678
+    protocol: TCP
+    name: http
+  selector:
+    app: fallback-svc
+' | kubectl apply -f -
+deployment.apps/fallback-svc created
+service/fallback-svc created
```

Comment:
Can you split out the result from the codeblock so that it's easily copyable? E.g:

Result:
```bash
deployment.apps/fallback-svc created
service/fallback-svc created
```